specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/ManagedInstanceVulnerabilityAssessments.json

```diff
@@ -0,0 +1,378 @@
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "name": "managedInstanceName",
+            "in": "path",
+            "description": "The name of the managed instance for which the vulnerability assessment is defined.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vulnerabilityAssessmentName",
+            "in": "path",
+            "description": "The name of the vulnerability assessment.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "VulnerabilityAssessmentName",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully removed the managed instance vulnerability assessment."
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidStorageAccountName - The provided storage account is not valid or does not exist.\n\n * 400 InvalidStorageAccountCredentials - The provided storage account shared access signature or account storage key is not valid.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found"
+          }
+        },
+        "x-ms-examples": {
+          "Remove a managed instance's vulnerability assessment": {
+            "$ref": "./examples/ManagedInstanceVulnerabilityAssessmentDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagedInstanceVulnerabilityAssessmentProperties": {
+      "description": "Properties of a managed instance Vulnerability Assessment.",
+      "required": [
+        "storageContainerPath"
+      ],
+      "type": "object",
+      "properties": {
+        "storageContainerPath": {
+          "description": "A blob storage container path to hold the scan results (e.g. https://myStorage.blob.core.windows.net/VaScans/).",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
+        },
+        "storageContainerSasKey": {
+          "description": "A shared access signature (SAS Key) that has write access to the blob container specified in 'storageContainerPath' parameter. If 'storageAccountAccessKey' isn't specified, StorageContainerSasKey is required.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
+        },
+        "storageAccountAccessKey": {
+          "description": "Specifies the identifier key of the vulnerability assessment storage account. If 'StorageContainerSasKey' isn't specified, storageAccountAccessKey is required.",
```

Comment:
seems confusing.
there is no "vulnerability assessment storage account".

Maybe use: "storage account for vulnerability assessment scan results"?